### ISSUE TITLE
flutter_fimber' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50

### ISSUE THREAD

**Issue by adrianvintu** [magillus/flutter-fimber#127]
Thank you very your awesome work!

Can you please help with a build issue?

When compiling with

```
    dependencies {
        classpath 'com.android.tools.build:gradle:7.3.1'
        ....
```

I get the following exception

```
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':flutter_fimber' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
```

Pubspec.xml
```
  fimber: ^0.6.6
  fimber_io: ^0.6.6
  flutter_fimber: ^0.6.5
```

The workaround is to use an older gradle `classpath 'com.android.tools.build:gradle:7.2.1'`

Can you please update flutter_fimber to use a newer kotlin gradle?

**Comment by magillus** [comment]
Thank you for giving me a hint on that. I try to take care of it soon

**Comment by magillus** [comment]
I might get this resolved in https://github.com/magillus/flutter-fimber/compare/main...upgrade.pluging still Work In Progress

**Comment by adrianvintu** [comment]
Thank you @magillus! There is no hurry on my side, I use 7.2.1 to compile anyway. Wish you a happy new year!

**Comment by josh-burton** [comment]
@magillus wondering if there is any update here?

**Comment by magillus** [comment]
I think this is resolved now with 0.7.0 update.
